### PR TITLE
naughty: Add pattern for SELinux hostnamectl denial

### PR DESCRIPTION
**Pull request by jelly**
Known issue #2291
Downstream report for RHEL 34: https://bugzilla.redhat.com/show_bug.cgi?id=1991444

### CODE REVIEW

**Comment by jelly** [comment]
I'm not 100% sure if this is correct, is there a way for me to locally test this?

File: naughty/fedora-34/2291-selinux-hostnamectl [hunk 1]
```diff
@@ -0,0 +1 @@
+avc:  denied  { add_name } for  * comm="systemd-hostnam" name="*" scontext=system_u:system_r:systemd_hostnamed_t:s0 tcontext=system_u:object_r:init_var_run_t:s0 tclass=dir permissive=0
```
Comment (marusak) [inline naughty/fedora-34/2291-selinux-hostnamectl:1]: Two things some to mind:
1. If you look at other avcs we have, it is always a good idea to stick to the pattern. So start with `* type=1400 audit(*): avc:  denied`... (Back in the day this hit us, as some pattern did not work - so might this as well, as it does not start with `*`)
2. You can test your pattern with `tests-policy`

How to use test policy:
1. Put output from test into file
2. `./tests-policy -o context < file_name` where context is for example `fedora-34` and `file_name` is the one, where you put output of the test

**Comment by martinpitt** [comment]
https://github.com/cockpit-project/cockpit/pull/16213 does not seem to actually need this, it went green. Filing the bugzilla was good of course, it is a bug -- it just doesn't seem to manifest in that particular test setup? So should we just close this?

**Comment by jelly** [comment]
Yes, I'll close it as tests succeed.

**Comment by marusak** [comment]
We see it now for example [here](https://logs.cockpit-project.org/logs/pull-16213-20210811-102834-7b4744c3-fedora-34/log.html#274-2), reopenning.

**Review by marusak — APPROVED** [review]
The pattern works. I wanted to complain about it not following the same pattern as rest of avc naughties , but then noticed, they already don't follow one pattern :)
Thanks!

**Comment by marusak** [comment]
I triggered `fedora-34@bots#2292` context in https://github.com/cockpit-project/cockpit/pull/16213 for testing, hopefully it hits it.

@jelly FYI - we can test also across projects, so we can do something like `fedora-34@cockpit-project/cockpit` for example in bots PR, and it will test current `main` in `cockpit-project/cockpit` with bots from the given PR. We can also do it the other way around, which is what I just did - testing bots PR in other project PR.

**Pull request merged**